bot/exts/utilities/wikipedia.py

```diff
@@ -87,7 +87,7 @@ async def wikipedia_search_command(self, ctx: commands.Context, *, search: str)
             embed.set_thumbnail(url=WIKI_THUMBNAIL)
             embed.timestamp = datetime.utcnow()
             await LinePaginator.paginate(
-                contents, ctx, embed
+                contents, ctx, embed, restrict_to_user=ctx.author
```

Comment:
This works well, but doesn't need to be split up like this. I'm aware you didn't do this yourself, but lets make it better while we're here. The style we currently go with is this

`await LinePaginator.paginate(contents, ctx, embed, restrict_to_user=ctx.author)`